### ISSUE TITLE
Failed to get visualizer process ID after launch (MAC OS 10.15.7) 

### ISSUE THREAD

**Issue by marson666** [Unity-Technologies/com.unity.perception#526]
Failed to get visualizer process ID after launch. Note that this does not necessarily mean the tool did not launch successfully. However, running the visualizer again will now create a new instance of the process.
UnityEngine.UnitySynchronizationContext:ExecuteTasks () (at /Users/bokken/buildslave/unity/build/Runtime/Export/Scripting/UnitySynchronizationContext.cs:107)

![image](https://user-images.githubusercontent.com/26239617/189307745-cc28ab2e-68d9-4919-abeb-7ea283e8aff0.png)

I have re-installed protobuf<3.20, but it still not work

**Comment by aryan-mann** [comment]
Hey @marson666! The visualizer is an experimental feature that depends on the experimental Python Scripting package. We've seen incompatibilities when trying to use it with the latest Python Scripting package.

I'd recommend running the visualizer from outside Unity by following Step 1 and Step 2 (Public Version) from the [com.unity.cv.datasetvisualizer](https://github.com/Unity-Technologies/com.unity.cv.datasetvisualizer) repo. After installing the python package, you can run the `datasetvisualizer` command in your terminal and a browser window with the visualizer should pop-up. Let me know if you have issues with this alternate method.

**Comment by aryan-mann** [comment]
Hey @marson666! The Perception team just released 🎉 Perception 1.0 🎉, a major update to the toolset! With this new release, we have deprecated the dataset visualizer in favor of a more readable output format called SOLO and a python toolchain for visualization and processing called pysolotools. More details here: [Perception 1.0: Expanding the Open Source Toolbox for Synthetic Data](https://blog.unity.com/technology/perception-open-source-toolbox-for-synthetic-data)

We recommend hopping on over to using SOLO + pysolotools instead! It should be as easy as just changing the output format and installing the pysolotools python package.

Closing this for now, feel free to open it back up if there are any issues.
